scripts/local/docker-compose.yml

```diff
@@ -126,6 +126,10 @@ services:
       - PCS_CORS_WHITELIST
       - PCS_APPLICATION_SECRET
       - PCS_SEED_TEMPLATE
+      - PCS_OFFICE365_CONNECTION_URL
+      - PCS_SOLUTION_NAME
+      - PCS_SUBSCRIPTION_ID
+      - PCS_ARM_ENDPOINT_URL
```

Comment:
we need similar change for `telemetry` java service:
      - PCS_AZUREBLOB_CONNSTRING
      - PCS_ACTION_EVENTHUB_CONNSTRING
      - PCS_ACTION_EVENTHUB_NAME
      - PCS_LOGICAPP_ENDPOINT_URL
      - PCS_SOLUTION_WEBSITE_URL

Reply:
Added above.